From fc5dc53397677464099e80629e785a25718bf5ec Mon Sep 17 00:00:00 2001
From: Xavier Hahn <xavier.hahn@gmail.com>
Date: Tue, 23 Mar 2021 22:52:27 +0100
Subject: [PATCH] Set the scope to the configuration properties

The configuration properties' scope define which level of configuration can override the settings.
The default is window which meams that workspace configuration can override them.
---
 package.json | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/package.json b/package.json
index b634a30..6eb59be 100644
--- a/package.json
+++ b/package.json
@@ -65,26 +65,31 @@
         "matlab.matlabpath": {
           "type": "string",
           "default": null,
+          "scope": "machine",
           "description": "The path to the matlab executable."
         },
         "matlab.mlintpath": {
           "type": "string",
           "default": null,
+          "scope": "machine",
           "description": "The path to the mlint executable"
         },
         "matlab.lintOnSave": {
           "type": "boolean",
           "default": true,
+          "scope": "window",
           "description": "Run the linting on save of file."
         },
         "matlab.linterEncoding": {
           "type": "string",
           "default": "utf8",
+          "scope": "window",
           "description": "This sets the encoding for the linting, default is utf8. Use if your console uses a different encoding."
         },
         "matlab.linterConfig": {
           "type": "string",
           "default": null,
+          "scope": "machine",
           "description": "This allows to pass a configuration settings file for the mlint linter. Enter the full path for the linter settings file to use."
         }
       }
